src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -19,12 +23,18 @@
 
 @RestController
 @RequestMapping("/api/books")
+@Api
 public class BookController {
 
     @Autowired
     private BookRepository bookRepository;
 
     @GetMapping
+    @ApiOperation(value = "List all Books")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Books found"),
+            @ApiResponse(code = 404, message = "Books not found")
```

Comment:
This method returns the list of books, so i dont think it will return 404.
I think it returns an empty list.
What do you think?